ui/src/design/ButtonBar/index.tsx

```diff
@@ -11,22 +11,12 @@ export const ButtonBar: React.FC<HTMLAttributes<HTMLDivElement>> = ({
     {...props}
     className={twMergeClsx(
       "[&>*]:rounded-none",
-      "[&>*]:mr-[-1px]",
+      "[&>*]:-mr-px",
       "[&>*]:active:outline-none",
       "[&>*]:border [&_button]:focus:ring-0 [&_button]:focus:ring-offset-0",
       "[&>*]:active:ring-0 [&_button]:active:ring-offset-0",
 
-      "[&>button]:rounded-none",
```

Comment:
I removed that block, as some of the lines defined the same behavior that was already covered in the first lines. Some of the lines (I can not recall which which lines :) ) were somewhat wrong in their syntax and produced an "invalid CSS class name" error when building the app.